src/shared/tpm2-util.c

```diff
@@ -5792,7 +5792,8 @@ int tpm2_seal_data(
                 const Tpm2Handle *encryption_session,
                 const TPM2B_DIGEST *policy,
                 struct iovec *ret_public,
-                struct iovec *ret_private) {
+                struct iovec *ret_private,
+                bool uses_pin) {
```

Comment:
we generally want return values to be at the end of the parameter lists. Hence please move this new arg before the first `ret_xyz` parameter (which is how we name return params)